plugin.video.roosterteeth/addon.xml

```diff
@@ -1,13 +1,15 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.roosterteeth"
        name="Rooster Teeth"
-       version="1.3.4"
-       provider-name="Skipmode A1, Abolfazl, Divingmule">
+       version="1.3.5"
+       provider-name="Skipmode A1">
   <requires>
-    <import addon="xbmc.python"                 version="2.14.0"/>
-	<import addon="script.module.beautifulsoup" version="3.0.8"/>
-	<import addon="script.module.requests"      version="2.4.3"/>
-    <import addon="plugin.video.youtube"        version="5.1.7"/>
+    <import addon="xbmc.python"                     version="2.14.0"/>
+    <import addon="script.module.beautifulsoup4"    version="4.5.3"/>
```

Comment:
@skipmodea1 ERROR: Version mismatch for addon script.module.beautifulsoup4. Required: 4.5.3, Available: 4.3.2